x-pack/plugins/security/index.js

```diff
@@ -37,6 +43,15 @@ export const security = (kibana) => new kibana.Plugin({
         hostname: Joi.string().hostname(),
         port: Joi.number().integer().min(0).max(65535)
       }).default(),
+      rbac: Joi.object({
+        application: Joi.string().default('kibana').regex(
```

Comment:
Maybe `rbac.application` makes more sense when you're more familiar with the privileges APIs, but I feel like something like `rbac.applicationId` would be clearer to someone like me who knows less about those APIs.

Reply:
`rbac.application` will hopefully be going away and we'll just use the Kibana index here, so I'll postpone this discussion in the hope that we don't need it.